src/license/isCKCdnSupportedByEditorVersion.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { destructureSemanticVersion } from '../utils/version/destructureSemanticVersion.js';
+import { isCKCdnTestingVersion, type CKCdnVersion } from '../cdn/ck/isCKCdnVersion.js';
+import { getLicenseVersionFromEditorVersion } from './getLicenseVersionFromEditorVersion.js';
+
+/**
+ * Checks if the CKEditor CDN is supported by the given editor version.
+ *
+ * @param version The CKEditor version.
+ * @returns `true` if the CDN is supported, `false` otherwise.
+ */
+export function isCKCdnSupportedByEditorVersion( version: CKCdnVersion ): boolean {
+	if ( isCKCdnTestingVersion( version ) ) {
+		return true;
+	}
+
+	const { major } = destructureSemanticVersion( version );
```

Comment:
```suggestion
	const { major } = dismantleSemanticVersion( version );
```
I was thinking that the word may be confused with JavaScript destructurization. But, tbh, I don't know if "dismantle" is better.